GroupDocs.Editor.MVC.Test/EditorControllerTest.cs

```diff
@@ -1,80 +1,55 @@
-﻿using GroupDocs.Editor.MVC.Controllers;
-using NUnit.Framework;
-using System.Web.Routing;
-using MvcContrib.TestHelper;
-using Huygens;
-using System;
-using System.Collections.Generic;
-using System.Text;
-using Newtonsoft.Json;
 using GroupDocs.Editor.MVC.Products.Common.Entity.Web;
+using GroupDocs.Editor.MVC.Products.Editor.Controllers;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Moq;
+using NUnit.Framework;
 
-namespace GroupDocs.Editor.MVC.Test
+namespace NUnitTestProject1
 {
-    [TestFixture]
-    public static class EditorControllerTest {
-       
+    public class EditorControllerTest
+    {
+        Mock<IConfigurationSection> mockServerConfSection;
+        Mock<IConfigurationSection> mockCommonConfSection;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field 'mockCommonConfSection' and declare it as a local variable in the relevant methods.](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-MVC/pullRequest?prid=5172828)